qaframework-bdd-tests/src/test/resources/features/refapp-2.x/17-Service/service.feature

```diff
@@ -0,0 +1,21 @@
+Feature:Service Management
+
+  Background:
+    Given user goes to the appointment scheduling app
+    When user goes to manage service page
+    Then user clicks on new service page
+    And user enters service details
+
+  @selenium
+  @service
+  Scenario: Service management
+    And user clicks on save button
+    And system checks the saved service
+    #edit section
```

Comment:
What do you think about `#edit section `vs `#edit service type`

Reply:
Resolved.